File: swift-and-obj-c/Piano Tiles/Piano Tiles/ViewController.m
//
//  ViewController.m
//  Piano Tiles
//
//  Created by Todd Duberstein on 7/18/14.
//  Copyright (c) 2014 DuberFam. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    HighScoreNumber = [[NSUserDefaults standardUserDefaults] integerForKey:@"HighScoreSaved"];
    
    HighScoreLabel.text = [NSString stringWithFormat:@"High Score: %li", (long)HighScoreNumber];
    
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: swift-and-obj-c/SPRITE KIT SWIFT STUFFS/FlappySwift/FlappySwift/ViewController.swift
//
//  ViewController.swift
//  FlappySwift
//
//  Created by Todd Duberstein on 2/25/19.
//  Copyright © 2019 DuberFam. All rights reserved.
//

import UIKit
import SpriteKit

extension SKNode {
    
    class func unarchiveFromFile(_ file : String) -> SKNode? {
        
        let path = Bundle
        
    }
    
}

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        if let scene = GameScene.unarchiveFromFile("GameScene") as? GameScene {
            
            let skView = self.view as! SKView
            skView.showFPS = true
            skView.showsNodeCount = True
            
        }
        
    }


}

